src/bindings/js/node/src/model_wrap.cpp

```diff
@@ -117,3 +118,12 @@ Napi::Value ModelWrap::get_outputs(const Napi::CallbackInfo& info) {
 
     return js_outputs;
 }
+
+Napi::Value ModelWrap::is_dynamic(const Napi::CallbackInfo& info) {
+    if (info.Length() > 0) {
+        reportError(info.Env(), "isDynamic() does not accept any arguments.");
+        return info.Env().Null();
+    }
+    bool result = _model->is_dynamic();
+    return Napi::Boolean::New(info.Env(), result);
+}
```

Comment:
ok @almilosz.

Reply:
ok @almilosz  Thanks for the feedback , on top of adding auto , I think I should also mark the variable as const since it's value wouldn't change within the scope of the function , I hope that is okay ?

Reply:
@qxprakash everything fine, but file should have an empty line at the end (github highlights this issue with red round icon)

Reply:
ok @vishniakov-nikolai should I add an empty line at the end of model_wrap.cpp ?